### DISCUSSION TITLE
How to pass `--parallel` when using pep-compliant `build`?

### DISCUSSION THREAD

**Discussion by rossburton** [pypa/setuptools#4733] [General]
If I invoke `setup.py build` directly I can pass `--parallel 64`. How can I do that when using `python3 -mbuild`?

**Comment by abravalheri** [comment]
You should be able to create a `setup.cfg` file with a `[build]` section and a `parallel = 32` option in it.

**Reply by rossburton** [reply]
I've also seen:
```
[build_ext]
parallell = 32
```
What is preferable?

**Reply by rossburton** [reply]
And from my reading of `build.py`, parallel has to be an `int`.

**Reply by abravalheri** [reply]
Yes, sorry, parallel is an integer.

`build_ext` should be fine (I think `build` just delegates to `build_ext`).
